waku/node/waku_node.nim

```diff
@@ -199,6 +200,11 @@ proc mountMetadata*(node: WakuNode, clusterId: uint32): Result[void, string] =
 
   return ok()
 
+## Waku Sharding
+proc mountSharding*(node: WakuNode, clusterId: uint32, shardCount: uint32): Result[void, string] =
```

Comment:
not sure i like this approach

Reply:
We had a stateless system before. I not sure we can do better than that.

Reply:
sometimes i see sharding as part of relay. but a light client without relay, would need it also. so guess it should be a "node" thing.